components/email_aliases/features.cc

```diff
@@ -5,8 +5,8 @@
 
 #include "brave/components/email_aliases/features.h"
 
-namespace email_aliases {
+namespace email_aliases::features {
 
-BASE_FEATURE(kEmailAliases, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(kEmailAliases, "EmailAliases", base::FEATURE_DISABLED_BY_DEFAULT);
```

Comment:
I think we should switch to `BASE_FEATURE(kEmailAliases, base::FEATURE_DISABLED_BY_DEFAULT);` after this is merged now https://github.com/brave/brave-core/pull/32041/